tsl/src/nodes/decompress_chunk/decompress_chunk.c

```diff
@@ -359,6 +359,87 @@ cost_decompress_chunk(Path *path, Path *compressed_path)
 	path->rows = compressed_path->rows * DECOMPRESS_CHUNK_BATCH_SIZE;
 }
 
+/*
+ * When limit queries use 'order by' and the first attribute of the order by is equivalent to the
+ * first attribute of the hypertable it can be beneficial to sort chunks individually. In that case
+ * ChunkAppend can be used and only the needed chunks are accessed (e.g., if the query has a small
+ * limit value, only the first chunk needs to be accessed in most cases).
```

Comment:
This is not about the first attribute but the first dimension.

Reply:
Thanks for the hint. The comment is changed.